bioptim/optimization/optimal_control_program.py

```diff
@@ -361,7 +361,7 @@ def __init__(
             self.update_parameters(parameters)
 
         # Declare the time to optimize
-        self._define_time(phase_time, objective_functions, constraints)
+        self.__define_time(phase_time, objective_functions, constraints)
```

Comment:
This is surprising, I think define time is supposed to be protected and not private... Can you compare with master?